utils/utils.py

```diff
@@ -16,6 +16,18 @@
 font = ImageFont.truetype('./assets/fonts/YuuPy_combined.ttf', 56)
 flag = Image.open('./assets/images/dialog/flag_overlay.png')
 textbox = Image.open('./assets/images/dialog/text_box.png')
+char_ribbon_alias = {
```

Comment:
I suggest that you send over the converted names so that you don't have to parse as much here

Reply:
Other than that it looks good

Reply:
I think a better option with be to strip `young_` from the string

Reply:
jin and yoshi are already processed, that should be removable, but I don't know what would be the best thing to do about the `young_` part.